bots/github/test-github

```diff
@@ -117,5 +117,28 @@ class TestGitHub(unittest.TestCase):
         count = api.get("/count")
         self.assertEqual(count, 1)
 
+class TestWhitelist(unittest.TestCase):
+    def setUp(self):
+        self.temp = tempfile.mkdtemp()
+
+    def tearDown(self):
+        shutil.rmtree(self.temp)
+
+    def testLoad(self):
+        filename = os.path.join(self.temp, "whitelist")
+        with open(filename, "w") as f:
+            f.write("   one   \n   two\nthree\n\n")
+        whitelist = github.whitelist(filename)
+        self.assertIn("one", whitelist)
+        self.assertIn("two", whitelist)
+        self.assertIn("three", whitelist)
+        self.assertNotIn("four", whitelist)
+        self.assertNotIn("", whitelist)
```

Comment:
I'd make this both simpler and more precise with `self.assertEqual(set(github.whitelist(filename)), set(["one", "two", "three"]))`. The first `set()` makes the test robust against possible implementation changes (like returning a list or a generator).